tests/buildTransaction.test.ts

```diff
@@ -0,0 +1,1470 @@
+
+    const saveTxButton = page.getByTitle("Save transaction");
+
+    await expect(saveTxButton).toBeDisabled();
+
+    const { operation_0 } = await selectOperationType({
+      page,
+      opType: "create_account",
+    });
+
+    await operation_0.getByLabel("Destination").fill(ACCOUNT_ONE);
+    await operation_0.getByLabel("Starting Balance").fill("1");
+
+    await expect(saveTxButton).toBeEnabled();
+    await saveTxButton.click();
+
+    const modal = page.locator(".Modal");
+
+    await expect(modal).toBeVisible();
+    await expect(page.locator(".ModalHeading")).toHaveText("Save Transaction");
+
+    await modal.getByLabel("Name", { exact: true }).fill("Transaction 1");
+    await modal.getByText("Save", { exact: true }).click();
+
+    await expect(modal).toBeHidden();
+  });
+
+  test.describe("Params", () => {
+    test("Happy path", async ({ page }) => {
+      const { paramsErrors } = getOperationsErrorsAndSuccessElements(page);
+      const fetchNextSeqButton = page.getByText("Fetch next sequence");
+
+      await expect(fetchNextSeqButton).toBeDisabled();
+
+      // Source account
+      await page.getByLabel("Source Account").fill(SOURCE_ACCOUNT);
+
+      // Sequence number
+      await expect(fetchNextSeqButton).toBeEnabled();
+      await fetchNextSeqButton.click();
+
+      // Base fee
+      await expect(page.getByLabel("Base fee")).toHaveValue("100");
+
+      // Mock fetch next sequence number
+      await page.route(`*/**/accounts/${SOURCE_ACCOUNT}`, async (route) => {
+        await route.fulfill({
+          status: 200,
+          contentType: "application/hal+json; charset=utf-8",
+          body: JSON.stringify(MOCK_SOURCE_ACCOUNT_RESPONSE),
+        });
+      });
+
+      await expect(page.getByLabel("Transaction Sequence Number")).toHaveValue(
+        SEQUENCE_NUMBER,
+      );
+
+      // Memo
+      await page
+        .getByTestId("memo-picker")
+        .locator(".RadioPicker__item")
+        .filter({ hasText: "Text" })
+        .click();
+
+      await expect(paramsErrors.getByRole("listitem")).toHaveText([
+        "Memo value is required when memo type is selected",
+      ]);
+      const memoValueInput = page.getByPlaceholder(
+        "UTF-8 string of up to 28 bytes",
+      );
+      await memoValueInput.fill("123");
+      await expect(paramsErrors).toBeHidden();
+
+      // Time bounds
+      const lowerTimeBoundsInput = page.getByPlaceholder(
+        "Lower time bound unix timestamp. Ex: 1479151713",
+      );
+      const upperTimeBoundsInput = page.getByPlaceholder(
+        "Upper time bound unix timestamp. Ex: 1479151713",
+      );
+
+      await lowerTimeBoundsInput.fill("1729270000");
+      await expect(lowerTimeBoundsInput.locator("+ div")).toHaveText(
+        "Fri, Oct 18, 2024, 16:46:40 UTC",
+      );
+
+      await upperTimeBoundsInput.fill("1729517340");
+      await expect(upperTimeBoundsInput.locator("+ div")).toHaveText(
+        /Mon, Oct 21, 2024, 13:29:00 UTC/,
```

Comment:
how come this one uses '/' regex

Reply:
Because the UI also includes the "Set to 5 min from now" text at the end of that div. I thought it would be confusing to add that text to the test. Regex makes it cleaner.

![image](https://github.com/user-attachments/assets/94f13e89-9467-48d5-a8c7-6f861de58ba4)